### DISCUSSION TITLE
Are there any one with merge privileges around?

### DISCUSSION THREAD

**Discussion by Nisden** [Handlebars-Net/Handlebars.Net#540] [General]
Its been a while since there have been any activity in the repository.
Are any of the contributors around?

I would very much like to progress with this pull request as its causing us issues.
https://github.com/Handlebars-Net/Handlebars.Net/issues/524

Alternately ill have to start publishing my own fork, and I rather not.

**Comment by StefH** [comment]
CC @zjklee & @rexm  ?

**Comment by Nisden** [comment]
@zjklee Can we have a new nuget release?

**Reply by oformaniuk** [reply]
@Nisden , I've released 2.1.3

**Reply by oformaniuk** [reply]
In general, I'm back online to the project after being away for some time due to personal reasons. In case you'd have capacity to contribute - it will go smoother next time 😉

**Reply by Nisden** [reply]
Thank you for the release! Hopefully I will have something else to commit back.

And no worries, it's open source so everything is best effort :)
